### DISCUSSION TITLE
[V3] Wire $refresh doesn't trigger loading states

### DISCUSSION THREAD

**Discussion by Cbrad24** [livewire/livewire#6555] [7. Bugs (archived)]
### Livewire version

v3.0.1

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a blank Livewire component:
`php artisan make:livewire TestRefresh`

2. Fill out the following class code:
```php
<?php

namespace App\Livewire;

use Livewire\Component;

class TestRefresh extends Component
{
    public function render()
    {
        sleep(1);
        return view('livewire.test-refresh');
    }
}
```

3. Fill out the following blade view code:
```php
<div>
    <div style="display: block; margin-bottom: 1rem;">
        <button wire:click="$refresh" wire:loading.remove>
            Test
        </button>
    </div>

    <div style="display: block;">
        <button wire:click="$refresh" wire:loading.attr="disabled">
            Test
        </button>
    </div>
</div>
```

4. Observe that clicking the buttons does fire off a $refresh command, however the loading states seem to be ignored when issued by a $refresh.

### Additional information

Came across this issue while doing a V2 to V3 port of my code, so the following code definitely used to work on V2:
```php
<button wire:loading.attr="loading" wire:target="$refresh" wire:click="$refresh">
  <Refresh Icon Here />
</button>
```

Clicking the button definitely still refreshes the page, but it doesn't append the "loading" attribute to my button anymore.

Thanks for the amazing framework 👍🏼

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by sparklem-agency** [comment]
same problem

**Comment by Cbrad24** [comment]
Bumping as still broken in the newest builds.

**Comment by RalpinMcDjango** [comment]
Still seems to be an issue on the `3.6.0` release

**Comment by ghabriel25** [comment]
Workaround https://github.com/livewire/livewire/discussions/9303#discussioncomment-12930224

**Reply by danie-ramdhani** [reply]
see: https://github.com/livewire/livewire/discussions/9303#discussioncomment-12932709
